### ISSUE TITLE
Models failed to be loaded

### ISSUE THREAD

**Issue by kaiquewdev** [adeshpande3/LSTM-Sentiment-Analysis#20]
NotFoundError (see above for traceback): Key rnn/basic_lstm_cell/kernel not found in checkpoint
	 [[Node: save_4/RestoreV2_4 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save_4/Const_0_0, save_4/RestoreV2_4/tensor_names, save_4/RestoreV2_4/shape_and_slices)]]

**Comment by kaiquewdev** [comment]
On both notebooks, how this can be solved?

**Comment by kaiquewdev** [comment]
Amazing work, thank you for share.

**Comment by adeshpande3** [comment]
Yeah, so that's an issue with the Tensorflow version. It works with Tensorflow version 1.1 but not 1.2 and above. Sorry, haven't had time to figure out why, but I'd recommend either downgrading or searching through the Tensorflow documentation to see what changes have been made to model loading.
